### PR TITLE
Add juniper.bot translation

### PR DESCRIPTION
**Pull request by MrLivixx**

### CODE REVIEW

File: src/Presence/juniperbot.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "juniperbot.mainpage": {
+      "message": "Main page",
+      "description": "Home page of the site"
+    },
+    "juniperbot.serverdash": {
+      "message": "In dashboard",
+      "description": "Guild: {name}"
+    },
```
Comment (Bas950) [inline src/Presence/juniperbot.json:9]: description correct?

File: src/Presence/juniperbot.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "description": "Page with bot statistics"
+    },
+    "juniperbot.usercard": {
+      "message": "Changes the rank card",
+      "description": "Rating card editing page"
+    },
+    "juniperbot.feedback": {
+      "message": "Reading:",
+      "description": "Reads the page"
+    }
```
Comment (Bas950) [inline src/Presence/juniperbot.json:33]: Not needed, you can use "general.reading" as the general.json file is always counted too.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by MrLivixx** [comment]
Fixed

**Comment by Bas950** [comment]
Aight looking good now, please link the pr of the presence update in here when ready

**Comment by MrLivixx** [comment]
> Aight looking good now, please link the pr of the presence update in here when ready

https://github.com/PreMiD/Presences/pull/2707

File: src/Presence/juniperbot.json [hunk 3]
```diff
@@ -30,5 +34,17 @@
     "juniperbot.usercard": {
       "message": "Changes the rank card",
       "description": "Rating card editing page"
+    },
+    "juniperbot.terms": {
+      "message": "Reads the Terms of use",
+      "description": "Reads the Terms of use"
+    },
```
Comment (Bas950) [inline src/Presence/juniperbot.json:41]: Use "general.reading" + "general.terms"

File: src/Presence/juniperbot.json [hunk 4]
```diff
@@ -30,5 +34,17 @@
...
+    "juniperbot.privacy": {
+      "message": "Reads the privacy Policy",
+      "description": "Reads the privacy Policy"
+    },
```
Comment (Bas950) [inline src/Presence/juniperbot.json:45]: Use "general.reading" + "general.privacy"

File: src/Presence/juniperbot.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
+{
+    "juniperbot.mainpage": {
+      "message": "Main page",
+      "description": "Home page of the site"
+    },
+    "juniperbot.leaderboard": {
+      "message": "Main page",
+      "description": "Home page of the site"
+    },
```
Comment (Bas950) [inline src/Presence/juniperbot.json:9]: Duplicate strings?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by MrLivixx** [comment]
Fixed it

File: src/Presence/juniperbot.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+      "message": "Looking at bot's statistics",
+      "description": "Page with bot's statistics"
+    },
+    "juniperbot.usercard": {
+      "message": "Customizing rank card",
+      "description": "Rating card customization page"
+    },
+    "juniperbot.cookies": {
+      "message": "Reads about the site's cookies",
+      "description": "Reads about the site's cookies"
```
Comment (Bas950) [inline src/Presence/juniperbot.json:40]: Reading about sounds better.

File: src/Presence/juniperbot.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+      "message": "Watching server's leaderboard:",
+      "description": "Guild leaderboard page"
+    }, 
+    "juniperbot.serverdash": {
+      "message": "In dashboard",
+      "description": "Server configuration page"
+    },
+    "juniperbot.serverdashname": {
+      "message": "Guild: {name}",
+      "description": "Guild name"
```
Comment (Bas950) [inline src/Presence/juniperbot.json:16]: use {0} instead

File: src/Presence/juniperbot.json [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+    },    
+    "juniperbot.donate":{
+      "message": "Reading information about donation plans",
+      "description": "Page with information about donation plans"
```
Comment (Bas950) [inline src/Presence/juniperbot.json:20]: about **the** donation plans

**Review by Bas950 — CHANGES_REQUESTED** [review]
After this you should be good to go

**Pull request merged**